src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/WebhookMessagePollingService.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.pay.webhooks.deliveryqueue.managed;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.context.internal.ManagedSessionContext;
```

Comment:
The fact this has `internal` in the package name hierarchy concerns me. We do not wish to be relying on internal implementation details.